client/src/components/PortalMetadataDownload.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { usePortalMetadataOnly } from 'hooks/usePortalMetadataOnly'
+import { DownloadModal } from 'components/DownloadModal'
+
+export const PortalMetadataDownload = () => {
+  const { portalMetadataComputedFile, isPortalMetadataOnlyAvailable } =
+    usePortalMetadataOnly()
+
+  return (
+    <>
+      {portalMetadataComputedFile && (
+        <DownloadModal
+          label="Get All Sample Metadata"
+          publicComputedFile={portalMetadataComputedFile}
+          disabled={!isPortalMetadataOnlyAvailable}
+        />
```

Comment:
I find it a bit curious that we pass the disabled prop here if we are just going to not render the button at all when `portalMetadataComputedFile` is not available. I think for now, without creating a much bigger refactor I would just render a regular button with the same label and make it disabled if `isPortalMetadataOnlyAvailable` if false. Also, I would add a brief comment explaining why.

Reply:
Thank you for your input - added `initialPublicComputedFile` to `useEffect`'s dependencies when setting `PublicComputedFile` in `useDownloadModal`, and removed the checks below accordingly 👍 (at 977d9757c1ee3ac21c4199c9b12868125e41342b)!